### PR TITLE
Fix #1331-Added option to zip a folder.

### LINKED ISSUE #1331: Adding option to zip a folder.
**Actual Behaviour**

There is no option to zip a folder.

**Expected Behaviour**

On performing longselect operation on a folder an option should be provided in the menu to zip the folder. Zipping a folder could be useful in sharing etc.

**Steps to reproduce it**

Perform long press operation on any folder. There is no option to zip the folder in the menu.

**Screenshots of the issue**

screenshot_2017-10-12-18-21-17-245_org fossasia phimpme

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sauravvishal8797**
Fix #1331 

Changes: Option is added to carry out the zip operation of a folder.

### CODE REVIEW

**Comment by sauravvishal8797** [comment]
@mohitmanuja Yeah. Will avoid it from next time.

**Comment by mayank8318** [comment]
@sauravvishal8797 Instead of pressing the Update button on GitHub, `rebase` and `force push` from the terminal to avoid the merge commits.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 1]
```diff
@@ -1998,6 +2024,80 @@ public void run() {
         }
     }
 
+    private class ZipAlbumTask extends AsyncTask<Void, Integer, Void>{
+
+        @Override protected void onPreExecute() {
+            super.onPreExecute();
+            progressDialog.setTitle(R.string.zip_action);
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Notification handler class can be used to show the Zipping progress in the notification panel

Comment (sauravvishal8797) [reply app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: Yeah. Will do that.

**Comment by anantprsd5** [comment]
Please resolve conflicts and update the PR with suggested changes. Thanks!

**Pull request merged**